modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx

```diff
@@ -277,6 +303,51 @@ const ActionForm = ({
 
 	return (
 		<>
+			{openIconsList && (
+				<ClayModal
+					containerProps={{
+						className: 'add-icon-modal',
+					}}
+					observer={observerIconsList}
+					size="lg"
```

Comment:
Is it intentional that we are using a small modal? There is a lot of scrolling down in a three-column solution. Why not use a full screen modal? This may be a question for @ugeortiz

Note to @chestofwonders : When you add a new UI in PR, please include a screenshot or a movie in description. This is how it looks:
![image](https://github.com/liferay-frontend/liferay-portal/assets/5435511/bb79273d-459e-44e7-8198-3d9c92b38a07)